packages/app/features/home/searchBar.tsx

```diff
@@ -20,14 +21,16 @@ const SearchBar: React.FC<SearchBarProps> = ( {onButtonPress} ) => {
 
   return (
     <View style={styles.searchBar}>
-      <TextInput
-        style={styles.input}
-        value={searchText}
-        onChangeText={handleChange}
-        placeholder="Type word"
-        placeholderTextColor="#999"
-      />
-      <Button title="search" onPress={handlePress}/>
+      <Row>
```

Comment:
Honest question, what is Row doing here? 🤔

Reply:
Keeping the text imput and the search button side by side